hwilib/devices/bitbox02.py

```diff
@@ -445,7 +445,41 @@ def display_multisig_address(
         pubkeys: List[PubkeyProvider],
         addr_type: AddressType,
     ) -> Dict[str, str]:
-        raise NotImplementedError("BitBox02 multisig not integrated into HWI yet")
+        path_suffixes = set(p.deriv_path for p in pubkeys)
+        if len(path_suffixes) != 1:
+            # Path suffix refers to the path after the account-level xpub, usually /<change>/<address>.
+            # The BitBox02 currently enforces that all of them are the same.
+            raise BadArgumentError("All multisig path suffixes must be the same")
+
+        # Figure out which of the cosigners is us.
+        key_origin_infos = {}
+        keypaths = {}
+        for pk in pubkeys:
+            assert pk.extkey and pk.origin
+            key_origin_infos[pk.extkey.serialize()] = pk.origin
+            keypaths[pk.extkey.serialize()] = pk.get_full_derivation_path(0)
+
+        if addr_type == AddressType.SH_WPKH:
```

Comment:
@achow101 @instagibbs 

it seems `AddressType` is not the right type to pass to this function? I found out by trial and error which types to match, but `SH_WPKH` does not make much sense, there is no pubkeyhash involved here? Same for `WPKH` below.

What this function ideally would receive is an enum with `SH_WSH`, `WSH` and `SH` as script types.

Reply:
Yes this seems like an enum misnomer. I think it's still logically "ok", the important part is what is wrapped where, so not pubkeys but scripts. Probably want to file an issue to fix this anyways.